es-app/src/views/SystemView.cpp

```diff
@@ -186,15 +192,18 @@ bool SystemView::input(InputConfig* config, Input input)
 		{
 			// get random system
 			// go to system
-			setCursor(SystemData::getRandomSystem());
+			if (setCursor(SystemData::getRandomSystem()))
+				Sound::get(mScrollSound)->play();
 			return true;
 		}
 	}else{
 		if(config->isMappedLike("left", input) ||
 			config->isMappedLike("right", input) ||
 			config->isMappedLike("up", input) ||
-			config->isMappedLike("down", input))
-			listInput(0);
+			config->isMappedLike("down", input)) {
+			if (listInput(0))
```

Comment:
Are we playing the sound on scroll and on stopping the scroll? I believe this is run when the button is released - is this deliberate?